cmake/Reprojucer.cmake

```diff
@@ -1576,6 +1576,7 @@ function(jucer_export_target_configuration
     set(library_search_paths "")
     foreach(path IN LISTS _EXTRA_LIBRARY_SEARCH_PATHS)
       file(TO_CMAKE_PATH "${path}" path)
+      set(path "${CMAKE_SYSROOT}/${path}")
```

Comment:
`path` might originally be a relative path such as:
- `../../../Dependencies/LibPdBuild` (in [`Camomille.jucer`](https://github.com/pierreguillot/Camomile/blob/78afd531b321e1933d434b7fddb172eb5262e67c/Instrument/Camomile.jucer#L112))
- `../../External/libusb/lib` (in [`LaMoucheFolle.jucer`](https://github.com/benkuper/LaMoucheFolle/blob/f03d431a17c79433a0d14dd3858301d90cb66731/LaMoucheFolle.jucer#L160))
- `..\..\nanogui\build\Debug` (in [`PtxCreator.jucer`](https://github.com/Hurleyworks/PtxCreator/blob/cc3ac0388c376e5518a45f6d2ea2d0fd7e5c502d/PtxCreator.jucer#L34))

Doing `set(path "${CMAKE_SYSROOT}/${path}")` would convert it to a (maybe broken) kind of absolute path, even if `CMAKE_SYSROOT` is empty.

Reply:
I'll take a look at these.  If you have any other interesting ones, I'll add them to my Yocto meta-layer.

I'm treating the .jucer file as the origin of truth, and not modifying post CMake generation.  That said I'll look into `jucer_export_target_configuration`